acoustid/data/meta.py

```
@@ -62,16 +64,50 @@ def generate_meta_gid(values: dict[str, Any]) -> uuid.UUID:
     return uuid.uuid5(meta_gid_ns, content_hash_source)
 
 
+def find_meta_id(conn: FingerprintDB, gid: uuid.UUID) -> int | None:
+    query = sql.select(schema.meta.c.id).where(schema.meta.c.gid == gid)
+    return conn.execute(query).scalar()
+
+
 def find_or_insert_meta(
     conn: FingerprintDB, values: dict[str, Any]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)